### PR TITLE
Feat/bm components interventions

### PR DESCRIPTION
**Pull request by radireddy**
| Question         | Answer
| ---------------- | ---
| Branch?          | feat/baremetal-server-components
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-6938, #MANAGER-6939
| License          | BSD 3-Clause

## Description

Add intervention and tasks components

### CODE REVIEW

File: packages/manager/modules/bm-server-components/src/support/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import angular from 'angular';
```
Comment (JeremyDec) [inline packages/manager/modules/bm-server-components/src/support/index.js:1]: Missing import of `@ovh-ux/ui-kit` here

File: packages/manager/modules/bm-server-components/src/technical-details/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import component from './component';
+
+const moduleName = 'ovhManagerBmServerTechnicalDetailsTileComponent';
+
+angular
+  .module(moduleName, [
+    'oui',
+    'pascalprecht.translate',
+    ngOvhFeatureFlipping,
+    ngAtInternet,
```
Comment (JeremyDec) [inline packages/manager/modules/bm-server-components/src/technical-details/index.js:17]:
```suggestion
    ngAtInternet,
    'ngTranslateAsyncLoader',
```

File: packages/manager/modules/bm-server-components/src/tasks/index.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import angular from 'angular';
```
Comment (JeremyDec) [inline packages/manager/modules/bm-server-components/src/tasks/index.js:1]: Same here for `@ovh-ux/ui-kit` import

File: packages/manager/modules/bm-server-components/src/interventions/index.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import angular from 'angular';
+
```
Comment (JeremyDec) [inline packages/manager/modules/bm-server-components/src/interventions/index.js:2]: Missing imports of:

- @ovh-ux/ui-kit
- @ovh-ux/ng-translate-async-loader

File: packages/manager/modules/bm-server-components/src/tasks/tasks.class.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+  isReady() {
+    return this.status === TASK_STATUS.DONE;
+  }
+
+  isUpdating() {
+    return (
+      this.status === TASK_STATUS.DOING ||
+      this.status === TASK_STATUS.INIT ||
+      this.status === TASK_STATUS.TODO
```
Comment (varun257) [inline packages/manager/modules/bm-server-components/src/tasks/tasks.class.js:21]: `[TASK_STATUS.DOING, TASK_STATUS.INIT, TASK_STATUS.TODO].includes(this.status)`

File: packages/manager/modules/bm-server-components/src/tasks/template.html [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+        data-title=":: 'server_configuration_task_comment' | translate"
+    >
+        <span data-ng-bind="$row.comment"></span>
+    </oui-datagrid-column>
+    <oui-datagrid-column
+        data-title=":: 'server_configuration_task_status' | translate"
+    >
+        <span
+            class="oui-badge"
+            data-translate="{{ 'server_configuration_task_status_' + $row.status }}"
```
Comment (varun257) [inline packages/manager/modules/bm-server-components/src/tasks/template.html:24]: `"{{:: 'server_configuration_task_status_' + $row.status }}"`

File: packages/manager/modules/bm-server-components/src/support/translations/Messages_fr_FR.json [hunk 7]
```diff
@@ -0,0 +1,9 @@
+{
+  "bm_server_support_heading": "Support",
+  "bm_server_support_level": "Niveau de support",
+  "bm_server_support_open_tickets": "Tickets ouverts",
+  "bm_server_support_create_ticket": "Créer un ticket",
+  "bm_server_interventions": "Intervention en cours",
+  "problem_with_your_product": "Un probléme sur vos produits?",
+  "consult_travaux": "Consulter nos tâches Travaux"
```
Comment (varun257) [inline packages/manager/modules/bm-server-components/src/support/translations/Messages_fr_FR.json:8]:
```
bm_server_problem_with_your_product
bm_server_ consult_travaux
```
To be more consistent

File: packages/manager/modules/bm-server-components/src/tasks/tasks.class.js (hunk 5 above)
Comment (radireddy) [reply packages/manager/modules/bm-server-components/src/tasks/tasks.class.js:21]: This looks simple but takes more time. Create a array and than loop through the array to check for the element.

**Comment by ganeshkumar1989** [comment]
Seems to have the support tile commit included...

**Review by ganeshkumar1989 — APPROVED** [review]

**Review by varun257 — APPROVED** [review]

**Pull request merged**